examples/content/src/content/jcr_root/apps/core-components-examples/components/demo/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Demo component is still WIP, adding initially so that we can initially reference it in the page content examples rather than moving everything later.